src/test/java/com/wikia/webdriver/testcases/adstests/TestIVW2AnalyticsProvider.java

```diff
@@ -20,11 +19,17 @@ public void testIVW2AnalyticsProvider() throws IOException {
       String wikiName = (String) data[0];
       String article = (String) data[1];
       String ivw2Param = (String) data[2];
+
       String testedPage = urlBuilder.getUrlForPath(wikiName, article);
       driver.get(testedPage);
       String htmlSource = driver.getPageSource();
-      Assertion.assertTrue(htmlSource.contains(ivw2Param));
-      PageObjectLogging.log("IVW2", ivw2Param + " param is on the " + testedPage, true);
+
+      Boolean isParamOnPage = htmlSource.contains(ivw2Param);
+      PageObjectLogging.log(
+          "IVW2",
+          ivw2Param + " param on the page: " + String.valueOf(isParamOnPage),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Directly append the argument of String.valueOf(). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1153)